### PR TITLE
Add translation for file browser title and description.

### PR DESCRIPTION
**Pull request by tao-xinxiu**

### CODE REVIEW

File: app/resources/locales/locale-fr.json [hunk 1]
```diff
@@ -12,6 +12,17 @@
   "Browse User Space files, Upload, and Select one file": "Parcourir les fichiers de l'Espace Utilisateur, Transférer, et Sélectionner un fichier",
   "Browse, Upload Global Space files, and Select one folder": "Parcourir, Transférer les fichiers de l'Espace Global et Sélectionner un dossier",
   "Browse, Upload User Space files, and Select one folder": "Parcourir, Transférer les fichiers de l'Espace Utilisateur et Sélectionner un dossier",
+  "GLOBAL DataSpace File Browser": "Navigateur de Fichiers de l'Espace de Données GLOBAL",
+  "USER DataSpace File Browser": "Navigateur de Fichiers de l'Espace de Données UTILISATEUR",
+  "GLOBAL DataSpace": "l'Espace de Données GLOBAL",
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:17]: Espace de Données GLOBAL (no need for l' normally)

File: app/resources/locales/locale-fr.json [hunk 2]
```diff
@@ -12,6 +12,17 @@
   "Browse User Space files, Upload, and Select one file": "Parcourir les fichiers de l'Espace Utilisateur, Transférer, et Sélectionner un fichier",
   "Browse, Upload Global Space files, and Select one folder": "Parcourir, Transférer les fichiers de l'Espace Global et Sélectionner un dossier",
   "Browse, Upload User Space files, and Select one folder": "Parcourir, Transférer les fichiers de l'Espace Utilisateur et Sélectionner un dossier",
+  "GLOBAL DataSpace File Browser": "Navigateur de Fichiers de l'Espace de Données GLOBAL",
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:15]: Navigateur de Fichiers de l'espace de données GLOBAL (no internal capital letters)

File: app/resources/locales/locale-fr.json [hunk 3]
```diff
@@ -12,6 +12,17 @@
...
+  "USER DataSpace File Browser": "Navigateur de Fichiers de l'Espace de Données UTILISATEUR",
+  "GLOBAL DataSpace": "l'Espace de Données GLOBAL",
+  "USER DataSpace": "l'Espace de Données UTILISATEUR",
+  "Browse, Upload/Download files in": "Parcourir, Télécharger / Télécharger des Fichiers dans",
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:19]: "Parcourir, Télécharger / Transférer des Fichiers dans"

File: app/resources/locales/locale-fr.json (hunk 1 above)
Comment (fviale) [reply app/resources/locales/locale-fr.json:17]: Unless you use this string only at the end of a sentence, in which case you need an article. But in that case, would it not be also the case in the other languages ?

File: app/resources/locales/locale-fr.json [hunk 4]
```diff
@@ -12,6 +12,17 @@
...
+  "Select one": "Sélectionnez un",
+  "folder": "dossier",
+  "file": "fichier",
+  "as the variable value": "comme la valeur de variable",
+  "Global DataSpace is a shared storage on the server host where anyone can read/write files.": "L'Espace de Données Global est un stockage partagé sur l'hôte du serveur où n'importe qui peut lire / écrire des fichiers.",
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:24]: L'espace de données Global est un stockage partagé sur l'hôte du serveur dans lequel tout utilisateur peut lire / écrire des fichiers.

File: app/resources/locales/locale-fr.json [hunk 5]
```diff
@@ -12,6 +12,17 @@
   "Browse User Space files, Upload, and Select one file": "Parcourir les fichiers de l'Espace Utilisateur, Transférer, et Sélectionner un fichier",
   "Browse, Upload Global Space files, and Select one folder": "Parcourir, Transférer les fichiers de l'Espace Global et Sélectionner un dossier",
   "Browse, Upload User Space files, and Select one folder": "Parcourir, Transférer les fichiers de l'Espace Utilisateur et Sélectionner un dossier",
+  "GLOBAL DataSpace File Browser": "Navigateur de Fichiers de l'Espace de Données GLOBAL",
+  "USER DataSpace File Browser": "Navigateur de Fichiers de l'Espace de Données UTILISATEUR",
+  "GLOBAL DataSpace": "l'Espace de Données GLOBAL",
+  "USER DataSpace": "l'Espace de Données UTILISATEUR",
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:18]: I think I would prefer to keep USER as it is a keyword and it should not be translated

File: app/resources/locales/locale-fr.json [hunk 6]
```diff
@@ -12,6 +12,17 @@
...
+  "Browse, Upload/Download files in": "Parcourir, Télécharger / Télécharger des Fichiers dans",
+  "Select one": "Sélectionnez un",
+  "folder": "dossier",
+  "file": "fichier",
+  "as the variable value": "comme la valeur de variable",
+  "Global DataSpace is a shared storage on the server host where anyone can read/write files.": "L'Espace de Données Global est un stockage partagé sur l'hôte du serveur où n'importe qui peut lire / écrire des fichiers.",
+  "User DataSpace is a personal user data storage.": "L'Espace de Données Utilisateur est un stockage de données personnelles des utilisateurs.",
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:25]: L'espace de données USER est un système de stockage de fichiers personnel pour chaque utilisateur.

File: app/resources/locales/locale-pt.json [hunk 7]
```diff
@@ -12,6 +12,17 @@
   "Browse User Space files, Upload, and Select one file": "Procurar arquivos do Espaço do Usuário, Carregar e Selecionar um arquivo",
   "Browse, Upload Global Space files, and Select one folder": "Procurar, Carregue arquivos de Espaço Global e Selecione uma pasta",
   "Browse, Upload User Space files, and Select one folder": "Procurar, Carregue arquivos de Espaço do Usuário e Selecione uma pasta",
+  "GLOBAL DataSpace File Browser": "Navegador de Arquivos de Espaço de Dados GLOBAL",
+  "USER DataSpace File Browser": "Navegador de Arquivos de Espaço de Dados DO UTILIZADOR",
+  "GLOBAL DataSpace": "Espaço de Dados GLOBAL",
+  "USER DataSpace": "Espaço de Dados DO UTILIZADOR",
```
Comment (aminelouati) [inline app/resources/locales/locale-pt.json:18]: No internal capital letter here too

File: app/resources/locales/locale-es.json [hunk 8]
```diff
@@ -12,6 +12,17 @@
...
+  "USER DataSpace File Browser": "Navegador de Archivos de Espacio de Datos del USUARIO",
+  "GLOBAL DataSpace": "Espacio de Datos GLOBAL",
+  "USER DataSpace": "Espacio de Datos del USUARIO",
+  "Browse, Upload/Download files in": "Examinar, Cargar / Descargar archivos en",
+  "Select one": "Seleccione uno",
+  "folder": "carpeta",
+  "file": "expediente",
+  "as the variable value": "como el valor de la variable",
+  "Global DataSpace is a shared storage on the server host where anyone can read/write files.": "Espacio de Datos GLOBAL es un almacenamiento compartido en el servidor donde cualquiera puede leer / escribir archivos.",
+  "User DataSpace is a personal user data storage.": "Espacio de Datos del USUARIO es un almacenamiento de datos de usuario personal.",
```
Comment (aminelouati) [inline app/resources/locales/locale-es.json:25]: de Datos -> de datos

File: app/resources/locales/locale-pt.json (hunk 7 above)
Comment (tao-xinxiu) [reply app/resources/locales/locale-pt.json:18]: For "USER DataSpace", should it be "Espaço de dados do UTILIZADOR" or "Espaço de dados USER" ?

**Review by aminelouati — APPROVED** [review]

**Review by fviale — APPROVED** [review]

**Comment by amouhoub** [comment]
@tao-xinxiu  Translations (object keys) should be alphabetically sorted. As you can see there is already an order which was slightly broken lately.
Could you please re-sort them please ? (using an online tool)

File: app/resources/locales/locale-fr.json [hunk 9]
```diff
@@ -7,16 +7,13 @@
   "Are you sure you want to permanently delete the file": "Voulez-vous vraiment supprimer définitivement le fichier",
   "Are you sure you want to permanently delete the folder": "Voulez-vous vraiment supprimer définitivement le dossier",
   "as a zip archive": "en tant qu'archive zip",
+  "as the variable value": "comme la valeur de variable",
   "Automation Dashboard": "Automation Dashboard",
   "Browse Global Space files, Upload, and Select one file": "Parcourir les fichiers de l'Espace Global, Transférer, et Sélectionner un fichier",
-  "Browse User Space files, Upload, and Select one file": "Parcourir les fichiers de l'Espace Utilisateur, Transférer, et Sélectionner un fichier",
+  "Browse, Upload/Download files in": "Parcourir, Télécharger / Transférer des fichiers dans",
```
Comment (fviale) [inline app/resources/locales/locale-fr.json:13]: after looking at the result, you should add an article at the end:
"Parcourir, Télécharger / Transférer des fichiers dans l'"

**Pull request merged**